src/helpers/daily-data-providers/fitbit-active-calories-burned.ts

```diff
@@ -0,0 +1,24 @@
+import { fitbitCaloriesBurnedDataProvider, fitbitRestingCaloriesBurnedDataProvider } from '.';
+import { CombinedDataCollectionSettings, getCombinedDataCollectionSettings } from './combined-data-collection-settings';
+import { DailyDataQueryResult } from '../query-daily-data';
+import { canQuery } from './data-collection-helper';
+
+export default async function(startDate: Date, endDate: Date, combinedDataCollectionSettings?: CombinedDataCollectionSettings): Promise<DailyDataQueryResult> {
+    const canQueryResult = canQuery(combinedDataCollectionSettings ?? await getCombinedDataCollectionSettings(), 'Fitbit', ['Calories', 'CaloriesBMR'], true);
+
+    if (canQueryResult.v1.enabled) {
+        const [totalCaloriesResult, restingCaloriesResult] = await Promise.all([
+            fitbitCaloriesBurnedDataProvider(startDate, endDate),
+            fitbitRestingCaloriesBurnedDataProvider(startDate, endDate)
+        ]);
+
+        return Object.keys(totalCaloriesResult).reduce((activeCaloriesResult, dayKey) => {
+            if (dayKey in restingCaloriesResult) {
+                activeCaloriesResult[dayKey] = totalCaloriesResult[dayKey] - restingCaloriesResult[dayKey];
```

Comment:
wonder if we should do some kind of check here to make sure values aren't 0 or something or that we don't end up with a negative value. Maybe this could never happen, but data can be weird sometimes.

Reply:
The values from the total and resting calorie data providers should always be greater than 0.  They have checks for that.  However, it would be theoretically possible here for a resting calorie value to be greater than the total calorie value, so I've added a check to make sure we only return values that are greater than 0.